assets/dev/scss/frontend/widgets/icon-list.scss

```diff
@@ -68,6 +70,7 @@
 
 	.elementor-icon-list-icon {
 		display: flex;
+		position: relative;
```

Comment:
Why?

Reply:
This is required for the `top` property to work when we adjust the vertical offset position. Using margin / negative margin doesn't work with `flex-start` or `flex-end` positions.